jakarta.ls/src/test/java/org/eclipse/lsp4jakarta/snippets/JakartaSnippetRegistryTest.java

```diff
@@ -0,0 +1,185 @@
+package org.eclipse.lsp4jakarta.snippets;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Optional;
+
+import org.eclipse.lsp4jakarta.commons.JavaCursorContextKind;
+import org.eclipse.lsp4jakarta.commons.JavaCursorContextResult;
+import org.eclipse.lsp4jakarta.commons.ProjectLabelInfoEntry;
+import org.eclipse.lsp4jakarta.commons.snippets.ISnippetContext;
+import org.eclipse.lsp4jakarta.commons.snippets.Snippet;
+import org.eclipse.lsp4jakarta.commons.snippets.SnippetRegistry;
+import org.junit.Test;
+
+/**
+ * Test for JakartaEE snippet registry.
+ **/
+
+public class JakartaSnippetRegistryTest {
+	
+	SnippetRegistry registry = new SnippetRegistry();
+
+	@Test
+	public void haveJavaSnippets() {
+		assertFalse("Tests has Jakarta Java snippets", registry.getSnippets().isEmpty());
+	}
+	
+	/**
+	 * Jakarta Bean Validation snippets - @Email 
+	 */
+	@Test
+	public void beanValidationSnippetsTest() {
+        Optional<Snippet> beanValidationSnippet = findByPrefix("@Email", registry);
```

Comment:
There is a mixture of spaces and tabs in this file. Please convert them all to spaces so that it will display properly in different contexts.

Reply:
I guess consistent use of tabs is ok.